packages/react/index.js

```diff
@@ -1,6 +1,50 @@
 'use strict';
 
+function testMinificationUsedDCE() {
+  // use scoped variable for our initial test, in case
+  // 'top-level' mangling is not enabled.
+  const source = testMinificationUsedDCE.toString();
```

Comment:
I wanted this line to be inside try catch in case toString throws. I think this can happen in restrictive environments although not sure where I saw that.